src/blueprint/from_config.rs

```diff
@@ -456,6 +433,11 @@ fn update_http(
         let output_schema = to_json_schema_for_field(field, config);
         let input_schema = to_json_schema_for_args(&field.args, config);
 
+        // Short-circuiting validation for group_by
+        if !http.group_by.is_empty() && http.method != Method::GET {
```

Comment:
```suggestion
        if is_batch_request {
```
Create a common variable that check if the request once.

Reply:
The later suggestion invalidates this, I think.